samples/csharp_dotnetcore/81.skills-skilldialog/DialogRootBot/appsettings.json

```diff
@@ -9,5 +9,8 @@
       "AppId": "",
       "SkillEndpoint": "http://localhost:39783/api/messages"
     }
-  ]
+  ],
+  "runtimeSettings:skills": {
+    "AllowedCallers": [ "*" ]
+  }
```

Comment:
Is this right?! Shouldn't allowed callers be applied only to the skill bot, and not to the consumer bot?